app/assets/stylesheets/core/_variables.scss

```diff
@@ -34,14 +34,14 @@ $orangeyRose:              #FEB09B;
 $teal:                     #43B596;
 
 // brand
+$shaleDark:				#212121;
 $shale:           #4a5658;
 $shaleLight:      #9Ba5aa;
 $shaleLighter:    #D4D9DB;
 $shaleLightest:   #F9FBFB;
-$shaleKindaLight: #A5A7A8;
```

Comment:
Is there a reason to remove these? I frequently go for these colors and having these shorthands is useful.

Reply:
The reason is the naming is inconsistent. 

If you need ```#A5A7A8``` we can rename it to ```#shaleLighterer``` or wherever it falls on the spectrum.

Reply:
shaleKindaLight is darker than shaleLight, so naming it shaleLighterer wouldn't make sense.

Reply:
Right now we're not using it anywhere so we could remove it.